src/registry/router.ts

```diff
@@ -28,6 +29,8 @@ export function create(app: Express, conf: Config, repository: Repository) {
   };
 
   const prefix = conf.prefix;
+  const publishRateLimiter = createPublishRateLimiter(conf);
+  console.log('publishRateLimiter', publishRateLimiter);
```

Comment:
I guess this console.log is not intended to be here

Reply:
Yup will delete